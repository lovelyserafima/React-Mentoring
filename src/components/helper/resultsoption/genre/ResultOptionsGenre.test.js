import React from 'react';
import renderer from 'react-test-renderer';
import ResultsOptions from "./ResultsOptionsGenre";

test('ResultsOptionsSearch renders', () => {
    const component = renderer.create(<ResultsOptions />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});