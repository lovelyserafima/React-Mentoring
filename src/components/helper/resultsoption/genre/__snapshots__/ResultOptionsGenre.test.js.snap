// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResultsOptionsSearch renders 1`] = `
<div
  className="ResultOptionsStyles__ResultsOptionsWrapper-sc-2ncvx-0 DjEDX"
>
  <p
    className="ResultOptionsStyles__Display-sc-2ncvx-2 eInEox"
  >
    Films by 
     genre
  </p>
</div>
`;
